src/inference/dev_api/openvino/runtime/iasync_infer_request.hpp

```diff
@@ -184,6 +184,7 @@ class OPENVINO_RUNTIME_API IAsyncInferRequest : public IInferRequest {
      * @brief Check that all tensors are valid. Throws an exception if it's not.
      */
     void check_tensors() const override;
+    const std::shared_ptr<IInferRequest>& get_sync_infer_request() const;
```

Comment:
why do we need it? looks like WA

Reply:
I don't want to move sync infer request to protected section